### DISCUSSION TITLE
Audacity 3.6 Limiter and Compressor Settings

### DISCUSSION THREAD

**Discussion by Julie712** [audacity/audacity#6916] [Q&A]
Audacity 3.6 made some sweeping changes to the settings in both the Limiter and Compressor effects. Could someone please provide settings for the new versions that would mimic the default settings of the older version?

**Comment by petersampsonaudacity** [comment]
@saintmatthieu  - can you by any chance answer this query?

**Reply by saintmatthieu** [reply]
Hey @Julie712, thanks for opening this discussion, it is a sensible request, sorry I didn't think about it before.
I'm looking into this for the compressor. WRT the limiter, though, you can still download the old limiter from the [Audacity plugin page](https://plugins.audacityteam.org/nyquist-plugins/effect-plugins/dynamics-processing).

**Reply by Julie712** [reply]
Good information...thank you. Great to know that the old limiter is available as a plugin download. Could you also offer the original compressor as a plugin download? If not, the settings to mimic the original would be helpful as well. Appreciate your help!

**Reply by saintmatthieu** [reply]
Unfortunately, our native Audacity effects are not plugins and cannot be injected like plugins or Nyquist scripts can, sorry.

**Reply by Julie712** [reply]
I appreciate your time and energy responding to my questions. Thank you!

**Comment by saintmatthieu** [comment]
Hey @Julie712, thanks for opening this discussion, it is a sensible request, sorry I didn't think about it before.
WRT the limiter, you can still download the old limiter from the [Audacity plugin page](https://plugins.audacityteam.org/nyquist-plugins/effect-plugins/dynamics-processing).
For the compressor, the best I can give you is the following:
![image](https://github.com/user-attachments/assets/0bbc011f-3175-4eef-8cc9-26b7e6ff5339)
But even then, you might not be satisfied. Here is a comparison:
![image](https://github.com/user-attachments/assets/c6423316-e72f-4285-af29-cda61b3e825d)
You can see that
* at the beginning, the legacy compressor ramps in although the signal is well below the -12dB threshold,
* it won't react the same to impulses. The alternative is to increase the attack time of the new version, but then it will differ in other ways, e.g. you won't get that look-ahead effect visible before 1.1 and 2.3 seconds.

**Comment by saintmatthieu** [comment]
@Julie712 FYI we've decided to restore the old compressor and give it a new name. Whether it will be part of a patch to the 3.6 release or of 3.7 is yet to be decided.
https://github.com/audacity/audacity/issues/6982

**Comment by Julie712** [comment]
Wonderful, thank you! Just a quick add...any chance you could restore the original compressor to the original place and then rename the new one? That would streamline things for everyone who wants to continue using the old compressor in their macros, etc.

**Reply by saintmatthieu** [reply]
Well tried, but we want to give the new one the sweet spot :D

**Comment by Julie712** [comment]
Lol. Understood. And thank you!
